src/directives/routeOnClick.js

```diff
@@ -9,7 +9,7 @@ function routeOnClickFactory (Route, $location, $window, $timeout) {
       const LEFT_BUTTON = 0;
       const MIDDLE_BUTTON = 1;
 
-      if (element.tagName === 'A') {
+      if (element.tagName === 'A' || element[0].tagName === 'A') {
```

Comment:
this one is also interesting, it looks like the tests require it, BUT does the directive actually need it?

Reply:
Interesting... was the first condition even working? Maybe go full jQuery on this one:
`if (element.is('a') { ...`